prometheus/prometheus.go

```diff
@@ -85,3 +94,12 @@ func (obj *Prometheus) UpdateCheckApplyTotal(kind string, apply, eventful, error
 	metric.Inc()
 	return nil
 }
+
+// UpdatePgraphStartTime updates the mgmt_pgraph_start_time_seconds metric
+// to the current timestamp.
+func (obj *Prometheus) UpdatePgraphStartTime() error {
+	if obj == nil {
```

Comment:
I like this :) FYI for future, you can put this all one line line, it's legal golang and gofmt accepts it!